src/System.Security.Permissions/src/System/Security/ISecurityEncodable.cs

```diff
@@ -6,5 +6,7 @@ namespace System.Security
 {
     public partial interface ISecurityEncodable
     {
+        void FromXml(System.Security.SecurityElement e);
```

Comment:
In the implementation, it might be nice to not fully qualify `SecurityElement ` and instead just do `void FromXml(System.Security.SecurityElement e);` or the like.